modules/shared/src/test/scala/org/tessellation/schema/generators.scala

```diff
@@ -46,7 +46,10 @@ object generators {
     Gen.oneOf(Responsive, Unresponsive)
 
   val idGen: Gen[Id] =
-    nesGen(str => Id(Hex(str)))
+    Gen
+      .listOfN(128, Gen.alphaChar)
+      .map(_.mkString)
+      .map(str => Id(Hex(str)))
```

Comment:
I made this suggestion in another PR -- I believe `listOfN` can generate an empty list, which is what `nesGen` avoids. Some tests may break if an empty string is used.

What about a proper `Hex` generator? Then we can implement others in terms of it:
```
  
  def hexGen(n: Int): Gen[Hex] = Gen.stringOfN(n, Gen.hexChar).map(Hex(_))

  val peerIdGen: Gen[PeerId] = hexGen(128).map(PeerId(_))
  val idGen: Gen[Id] = hexGen(128).map(Id(_))
  val signatureGen: Gen[Signature] = Gen.chooseNum(140, 144).flatMap(hexGen).map(Signature(_))
```